src/schema/v2/homeView/HomeViewSection.ts

```diff
@@ -60,10 +59,13 @@ export const HomeViewSectionTypeNames = {
   HomeViewSectionViewingRooms: "HomeViewSectionViewingRooms",
 } as const
 
-const HomeViewGenericSectionInterface = new GraphQLInterfaceType({
+export const HomeViewGenericSectionInterface = new GraphQLInterfaceType({
   name: HomeViewSectionTypeNames.HomeViewSectionGeneric,
   description: "Abstract interface shared by every kind of home view section",
   fields: standardSectionFields,
+  resolveType: (value) => {
+    return value.type
+  },
```

Comment:
Ahh nice, I actually didn't realize interfaces also took `resolveType`  

(I think we've got dozens of these for unions and [only one](https://github.com/artsy/metaphysics/blame/7bd5e7a988f761185a2310b3132eca9533750ca0/src/schema/v2/artwork/artworkContextGrids/index.ts#L40-L63) for interfaces!)

I think that's what makes this possible — love it.